storage/filestore/dir.go

```diff
@@ -144,69 +166,157 @@ func (dir *Dir) Commit(ctx context.Context, file *os.File, ref storage.BlobRef)
 	return nil
 }
 
-// Open opens the file with the specified ref
-func (dir *Dir) Open(ctx context.Context, ref storage.BlobRef) (_ *os.File, err error) {
+// Open opens the file with the specified ref. It may need to check in more than one location in
+// order to find the blob, if it was stored with an older version of the storage node software.
+// In cases where the storage format version of a blob is already known, OpenSpecific() might be
+// a better choice.
```

Comment:
nit: I'd change `might be a better choice` to `will increase performance` or similar.

Reply:
It won't always increase performance. In some cases it will be the same. It might even always be a good choice, because it might not make as much sense in some contexts. I only want to remind the reader that OpenSpecific exists and might be a thing to do.